packages/components/addon/components/hds/dropdown/footer.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="hds-dropdown-footer" ...attributes>
```

Comment:
should we use a more BEM-y notation here? `hds-dropdown__footer` (no strong opinions, I see benefits also in keeping it as is, to differentiate from the other `hds-dropdown__***` sub-elements)

Reply:
Yeah, good call here. I think the element notation would make more sense here, indeed, so I've updated to `hds-dropdown__footer` as we don't expect this to be invoked on its own, we've only placed it in a separate file to allow splattributes.